benchmark_runner/grafana/perf/extract_dashboard.py

```diff
@@ -0,0 +1,19 @@
+import re, sys, json
+
+if len(sys.argv) != 3:
+    raise ValueError('Usage: python extract_dictionary.py input_file output_file')
```

Comment:
It's called extract_dashboard.  But you really should have it read from stdin and write to stdout, without any temporary files.

Reply:
You right, remove temporary file, read directly from stdin

`terraform apply -auto-approve | python3 extract_dashboard.py dashboard.json`